Description: Fix FTBFS on kfreebsd
 "#define BSD 1" also has to be un-commented in include/unixconf.h in order
 for nethack to be built successfully on kfreebsd (see
 0017-Debian-and-Linux-specifics-defined-in-unixconf.h.patch). Also, if
 0011-Pasi-Kallinen-s-patch-to-add-colors-to-inventory-ite.patch (i.e. the
 menucolors patch) is applied, defining MENU_COLOR_REGEX causes a FTBFS on
 kfreebsd (not sure why), so define it only on Linux for now.
Author: Vincent Cheng <Vincentc1208@gmail.com>
Last-Update: 2012-06-11

--- a/src/hacklib.c
+++ b/src/hacklib.c
@@ -3,6 +3,10 @@
 /* Copyright (c) Robert Patrick Rankin, 1991		  */
 /* NetHack may be freely redistributed.  See license for details. */
 
+#if defined(__FreeBSD_kernel__)
+#include <time.h>
+#endif
+
 /* We could include only config.h, except for the overlay definitions... */
 #include "hack.h"
 /*=
--- a/util/makedefs.c
+++ b/util/makedefs.c
@@ -4,6 +4,10 @@
 /* Copyright (c) Dean Luick, 1990.				  */
 /* NetHack may be freely redistributed.  See license for details. */
 
+#if defined(__FreeBSD_kernel__)
+#include <time.h>
+#endif
+
 #define MAKEDEFS_C	/* use to conditionally include file sections */
 /* #define DEBUG */	/* uncomment for debugging info */
 
--- a/win/tty/termcap.c
+++ b/win/tty/termcap.c
@@ -835,10 +835,6 @@
 
 #include <curses.h>
 
-#ifndef LINUX
-extern char *tparm();
-#endif
-
 #  ifdef COLOR_BLACK	/* trust include file */
 #undef COLOR_BLACK
 #  else
